src/UndockedRegFreeWinRT/UndockedRegFreeWinRT/dllmain.cpp

```diff
@@ -191,10 +195,36 @@ HRESULT WINAPI RoActivateInstanceDetour(HSTRING activatableClassId, IInspectable
     return S_OK;
 }
 
+VERSIONHELPERAPI IsWindowsVersionOrGreaterEx(WORD wMajorVersion, WORD wMinorVersion, WORD wServicePackMajor, WORD wBuildNumber)
+{
+    OSVERSIONINFOEXW osvi = { sizeof(osvi), 0, 0, 0, 0, {0}, 0, 0 };
```

Comment:
This can just be `OSVERSIONINFOEXW osvi = { sizeof(osvi) };`
which is nicer imho. Subjective. Gcc might warn. Visual C++ will not.
If there is a union in there, it might not zero it all, but I don't think you need it to anyway.